src/index.ts

```diff
@@ -6,16 +6,18 @@ import { analyze } from "./analyze.js";
 program
   .name("elmjs-inspect")
   .description("Analyze your elm.js file size with this tool.")
-  .version("1.1.0");
-
-program
-  .option(
+  .version("1.1.0")
+  .requiredOption(
```

Comment:
Summary seems core to the application and so I made it required, if there is a reason for it being optional (seems strange to me), then let me know and I can change back again.